docs/primer/how.md

```diff
@@ -108,14 +108,27 @@ Some schema fields refer to [codelists](../schema/codelists), to limit and stand
 
 In addition to the default format of JSON, you can convert and publish your OCDS data as Excel and CSV (comma-separated values) files.
 
-```{admonition} Publication formats: JSON and Excel/CSV
+````{admonition} Publication formats: JSON and Excel/CSV
 :class: hint
 
 While the OCDS schema is described using JSON Schema, OCDS data can be converted from its JSON format to tabular formats such as CSV files or spreadsheets. JSON is favored by developers because it uses human-readable text to exchange complex information, such as nested objects. It can contain large volumes of information and is particularly good at handling one-to-many relationships (such as multiple bids per tender notice).
 
-Tabular formats, such as CSV (or comma separated values) are commonly used in spreadsheets applications and other analysis tools. Many people are comfortable working with spreadsheets using tools like Excel. While JSON is the default format, a good publication will publish tabular formats as well both so that more users' needs can be satisfied.
+Tabular formats, such as CSV (or comma separated values) are commonly used in spreadsheet applications and other analysis tools. Many people are comfortable working with spreadsheets using tools like Excel. While JSON is the default format, a good publication will publish tabular formats both so that more users’ needs can be satisfied.
```

Comment:
```suggestion
Tabular formats, such as CSV (or comma separated values) are commonly used in spreadsheet applications and other analysis tools. Many people are comfortable working with spreadsheets using tools like Excel. While JSON is the default format, a good publication will publish tabular formats as well, so that more users’ needs can be satisfied.
```